app/Livewire/Project/Application/Backups.php

```
@@ -0,0 +1,30 @@
+use Livewire\Attributes\Locked;
+use Illuminate\Support\Facades\Gate;
+
+class Backups extends Component
+{
+    #[Locked]
+    public Application $application;
+    public Collection $databases;
+
+    public function mount(): void
+    {
+        abort_if(Gate::denies('view', $this->application), 403);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Gate' in method 'mount'. (undefined)

(StaticAccess)